functions/Set-DbaLogin.ps1

```diff
@@ -152,6 +160,8 @@ function Set-DbaLogin {
         [string[]]$Login,
         [Alias("Password")]
         [object]$SecurePassword, #object so that it can accept credential or securestring
+        [Alias("DefaulDB")]
```

Comment:
I only say this cause I was looking at typos and saw the tweet at the same time. You're missing a 't' in 'DefaultDB'
```suggestion
        [Alias("DefaultDB")]
```

Reply:
I wanted to make sure the spelling and casing of the parameter and its alias matched the rest of the project so I copy and pasted it from New-DbaLogin which evidently has the same typo.